bower.json

```diff
@@ -22,7 +22,14 @@
     "angular-sanitize": "^1.5.5",
     "firebase": "^2.4.2",
     "angularfire": "^1.2.0",
-    "isotope": "1.5.21",
-    "imagesloaded": "^4.1.0"
+    "imagesloaded": "^4.1.0",
+    "isotope": "^1.5.21"
+  },
+  "overrides": {
+    "font-awesome": {
+      "main": [
+        "css/font-awesome.css"
+      ]
+    }
   }
```

Comment:
Yep! knew that ...if you saw in the ticket comment I mention I tried that  and didn't work : glad u made it .